iac/terraform/aws/modules.tf

```diff
@@ -2,3 +2,23 @@ module "auth" {
   source      = "./auth"
   environment = var.environment
 }
+
+module "database" {
+  source     = "./dynamodb"
+  table_name = var.table_name
```

Comment:
Do not pass the table name from variable. It's a fixed name for each environment.

Reply:
Remove the variable and set a fixed name and prefix with the environment. [commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-b98a170a3603bd1524f7ec02d8baaac3ca60e30b0519c8d232baafc19edd1cb6)